apps/web/src/components/tx-flow/flows/SafenetTokenTransfers/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { useVisibleBalances } from '@/hooks/useVisibleBalances'
+import { type SafeBalanceResponse } from '@safe-global/safe-gateway-typescript-sdk'
+
+export const useTokenAmount = (selectedToken: SafeBalanceResponse['items'][0] | undefined) => {
+  const totalFiatAmount =
```

Comment:
I do not understand why we need to consider `decimals` for the `fiatBalance`.
The decimals should only matter for the `balance`.

Generally I feel like the maxAmount is a bit more complex. IMO it should consider the upper bound of e.g. 100.000$ in fiat.
But also consider how much unified balance the user has and how much is already used up by previous transfers.

Reply:
As we decided in our call, let's move this topic out of the happy path: https://github.com/safe-global/safenet/issues/290